### PR TITLE
added zeta in paddle elementwise.py

### PR DESCRIPTION
**Pull request by vandit98**
zeta #13408

### CODE REVIEW

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]
don't forget the unsupported ditypes decorator, uint16 and `bfloat16` are unsupported in all functions, and if there are more dtypes that you cant support, add them accordingly
```
@with_unsupported_device_and_dtypes(
    {"2.4.2 and below": {"cpu": ("uint16", "bfloat16")}}, backend_version
)
```
also the result is different than the ground truth, this is also a problem in the numpy implementation where this is derived from, so you will need to rethink this again
```
E           AssertionError:  the results from backend paddle and ground truth framework tensorflow do not match
E            [2.]!=[1.64493407] 
E           
E           
E           Falsifying example: test_zeta(
E               dtype_and_x=(['float64', 'float64'], [array([2.]), array([1.])]),
```

File: ivy/functional/backends/paddle/experimental/elementwise.py [hunk 1]
```diff
@@ -303,7 +303,21 @@ def zeta(
...
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    temp = ivy.logical_and(ivy.greater(x, 0), ivy.equal(ivy.remainder(x, 2), 0))
+    temp = ivy.logical_and(temp, ivy.less_equal(q, 0))
+    temp = ivy.logical_and(temp, ivy.equal(ivy.remainder(q, 1), 0))
+    inf_indices = ivy.logical_or(temp, ivy.equal(x, 1))
+    temp = ivy.logical_and(ivy.not_equal(ivy.remainder(x, 2), 0), ivy.greater(x, 1))
+    temp = ivy.logical_and(temp, ivy.less_equal(q, 0))
+    nan_indices = ivy.logical_or(temp, ivy.less(x, 1))
+    n, res = 1, 1 / q**x
```
[the remark this answers is not in the document]
Comment (vandit98) [reply ivy/functional/backends/paddle/experimental/elementwise.py]: yes, I agree 
I will make the changes.

**Comment by vandit98** [comment]
hi @MahmoudAshraf97 
Please review my changes. Test are passing.
Screenshot from 2023-04-03 13-52-22

I have used jax implementation and also we need to update numpy backend zeta function.
I will be happy to do it as well.
Thanks

File: ivy/functional/backends/paddle/experimental/elementwise.py [hunk 2]
```diff
@@ -346,14 +347,57 @@ def nextafter(
...
     q: paddle.Tensor,
     /,
     *,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+   with ivy.ArrayMode(False):
+        s,a=ivy.promote_types_of_inputs(x,q)
+        s_, a_ = paddle.unsqueeze(x, -1), paddle.unsqueeze(q, -1)
+        N = M = np.float64(8) if q.dtype == np.float32 else np.float64(16)
```
Comment (MahmoudAshraf97) [inline ivy/functional/backends/paddle/experimental/elementwise.py]: please review this

**Review by MahmoudAshraf97 — APPROVED** [review]
Good work

**Pull request merged**